Packs/ContentTesting/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### UnitTest
+
+- Change line 93 from      'args':{},                      'args': newargs, 
```

Comment:
if the change being made does not have any visible impact on the user, then the note should be 'Documentation and metadata improvements.', but i need to understand, what is the purpose of this change? and what is to support xsoar 8.2 you mentioned in the description